flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/assigners/ChunkSplitter.java

```diff
@@ -265,13 +269,13 @@ private static boolean isSplitColumnEvenlyDistributed(
         // only column is numeric and evenly distribution factor is less than
         // MAX_EVENLY_DISTRIBUTION_FACTOR will be treated as evenly distributed.
         final long approximateRowCnt = queryApproximateRowCnt(jdbc, tableId);
-        final double evenlyDistributionFactor =
+        final double actualEvenlyDistributionFactor =
                 calculateEvenlyDistributionFactor(min, max, approximateRowCnt);
         LOG.info(
                 "The evenly distribution factor for table {} is {}",
                 tableId,
-                evenlyDistributionFactor);
-        return evenlyDistributionFactor <= MAX_EVENLY_DISTRIBUTION_FACTOR;
+                actualEvenlyDistributionFactor);
```

Comment:
We can improve the log a bit to print the expected `evenlyDistributionFactor` as well, and tell users which mode is chosen.